tests/acceptance/features/apiSpaces/tag.feature

```diff
@@ -0,0 +1,80 @@
+
+
+  Scenario: Alice creates tags for resources in the project space
+    Given user "Alice" has shared a space "use-tag" to user "Brian" with role "viewer"
+    When user "Alice" creates the following tags for folder "folderMain" of space "use-tag":
+      | tag level#1                    |
+      | tag with symbols @^$#^%$@%!_+) |
+    Then the HTTP status code should be "200"
+    When user "Alice" sends PROPFIND request from the space "use-tag" to the resource "folderMain" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the "PROPFIND" response should contain a space "use-tag" with these key and value pairs:
+      | key     | value                                      |
+      | oc:tags | tag level#1,tag with symbols @^$#^%$@%!_+) |
+    When user "Alice" creates the following tags for file "folderMain/insideTheFolder.txt" of space "use-tag":
+      | fileTag |
+    Then the HTTP status code should be "200"
+    When user "Brian" sends PROPFIND request from the space "use-tag" to the resource "folderMain/insideTheFolder.txt" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the "PROPFIND" response should contain a space "use-tag" with these key and value pairs:
+      | key     | value   |
+      | oc:tags | fileTag |
+    When user "Alice" lists all available tags via the GraphApi
+    Then the HTTP status code should be "200"
+    And the user should have following tags:
+      | tag level#1                    |
+      | tag with symbols @^$#^%$@%!_+) |
+      | fileTag                        |
+    When user "Alice" lists all available tags via the GraphApi
+    Then the HTTP status code should be "200"
+    And the user should have following tags:
+      | tag level#1                    |
+      | tag with symbols @^$#^%$@%!_+) |
+      | fileTag                        |
+
+
+  Scenario: Alice creates tags for resources in the personal space
+    Given user "Alice" has created a folder "folderMain" in space "Alice Hansen"
+    And user "Alice" has uploaded a file inside space "Alice Hansen" with content "some content" to "file.txt"
+    When user "Alice" creates the following tags for folder "folderMain" of space "Alice Hansen":
+      | my tag    |
+      | important |
+    Then the HTTP status code should be "200"
+    When user "Alice" creates the following tags for file "file.txt" of space "Alice Hansen":
+      | fileTag                       |
+      | tag with symbol @^$#^%$@%!_+) |
+    Then the HTTP status code should be "200"
+    When user "Alice" sends PROPFIND request from the space "Alice Hansen" to the resource "folderMain" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the "PROPFIND" response to user "Alice" should contain a mountpoint "Alice Hansen" with these key and value pairs:
+      | key     | value            |
+      | oc:tags | my tag,important |
```

Comment:
what about combining those steps in to a single Then-step. e.g.
`@Then /^(?:file|folder|entry) "([^"]*)" of space "([^"]*)" should have the following tags for user "([^"]*)"$/`
then we would not need to repeat When-Then-And

Reply:
I'm not sure. I wanted to make the steps more transparent and show in the test that I use propfind request